cluster/addons/dns/skydns-rc.yaml.in

```diff
@@ -21,7 +21,7 @@ desiredState:
         dnsPolicy: "Default"  # Don't use cluster DNS.
         containers:
           - name: etcd
-            image: quay.io/coreos/etcd:v2.0.3
+            image: gcr.io/google-containers/etcd:v2.0.3
```

Comment:
We don't maintain this image, Core does. We should possibly keep it at quay unless Core wants to move it?